src/ol3googlemaps.js

```diff
@@ -64,12 +64,15 @@ olgm.OLGoogleMaps = function(options) {
   var watchVector = options.watchVector !== undefined ?
       options.watchVector : true;
 
+  var mapIconOptions = options.mapIconOptions !== undefined ?
+      options.mapIconOptions : {useCanvas: false};
```

Comment:
I think we should define a blank `{}` here instead of setting `useCanvas: false`.  It makes it easier for future addition, i.e. we wouln't have to think to add them here.